portal-impl/src/com/liferay/portal/security/auth/AuthVerifierPipeline.java

```diff
@@ -212,6 +222,32 @@ private synchronized void _removeAuthVerifierConfiguration(
 	private volatile URLPatternMapper<List<AuthVerifierConfiguration>>
 		_includeURLPatternMapper;
 
+	private static class AuthVerifierConfigurationCompartor
+		implements Comparator<AuthVerifierConfiguration> {
```

Comment:
Type in `Compartor`. Should be `Comparator`

Reply:
Good catch!